hermes/bin/src/runtime_extensions/hermes/cron/mod.rs

```diff
@@ -337,8 +346,6 @@ impl Ord for CronComponent {
 
 #[cfg(test)]
 mod tests {
-    use time::Duration;
```

Comment:
The `#[allow(clippy::expect_used)]` above this line needs to be removed as does the `.expect` that its masking.
`expect` is not allowed outside of unit tests.
The node is not to panic, and `expect` is a forced panic.
far better in this case to log an error, and return that the two things are not comparable.